src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.cs

```diff
@@ -1310,7 +1316,6 @@ private void Update(bool isIntermediate)
 			ScrollOffsets = new Point(_pendingHorizontalOffset, _pendingVerticalOffset);
 			InvalidateViewport();
 #endif
-
 			ViewChanged?.Invoke(this, new ScrollViewerViewChangedEventArgs { IsIntermediate = isIntermediate });
```

Comment:
fwiw, this doesn't match the UWP API. The property shouldn't have a public setter. This is unrelated to the PR. Let me know if you want a separate tracking issue for that.

Reply:
Which property please?

Reply:
IsIntermediate

Reply:
I see, you can make the setter `internal` then, as it will match UWP. It is a breaking change though, so you will need to update `PackageDiffIgnore.xml` accordingly. You can for now just make it internal and commit the change, and the build will tell you specifically how to update that file.

Reply:
I think this might be suitable for a separate PR since this is not part of `ViewChanging`. Going to open another issue